src/pages/Result.tsx

```diff
@@ -27,19 +30,58 @@ type Lyrics = {
     warped: Lyric[];
 };
 
+const MEME_CREATION_PROMPTS = [
+    'Highlight some lyrics you like, then press next.',
+    'Another prompt',
+];
+
+const renderMemeCreationTools = (stage: number, windowSelection: string) => {
+    switch(stage) {
+        case 0: {
+            return (
+                <>
+                    <MemeToolsPrompt>{MEME_CREATION_PROMPTS[stage]}</MemeToolsPrompt>
+                    <MemeToolsNextButton enabled={Boolean(windowSelection)}>
+                        <MemeToolsNextButtonText>Next</MemeToolsNextButtonText>
+                    </MemeToolsNextButton>
+                </>
+            );
+        }
+    }
+};
+
 const Result: React.FC<Props> = ({ resultId }) => {
     const [songData, setSongData] = useState<{
         title: string;
         artistName: string;
         lyrics: Lyrics;
     } | null>(null);
+    
+    const [memeCreationStage, setMemeCreationStage] = useState<number>(0);
```

Comment:
Take a peek at TS [Enums](https://www.typescriptlang.org/docs/handbook/enums.html) for handling types like this. 

They let you do things like 
```ts
enum Stage {
    PickImage,
    PickFont,
    ShareImage
}

switch(stage) {
    case Stage.PickImage:
        return <p>Pick your Image son</p>
    default:
        return <p>something has gone very wrong</p>;
}
```